client/src/main/python/slipstream/SlipStreamHttpClient.py

```diff
@@ -57,7 +57,7 @@ def _assemble_endpoints(self):
         self.runEndpoint = self.serviceurl + util.RUN_RESOURCE_PATH
         self.run_url = self.runEndpoint + '/' + self.diid
 
-        self.authnServiceUrl = self.serviceurl + '/'
+        self.authnServiceUrl = self.serviceurl + '/api/session'
```

Comment:
The `auth/login` URL is deprecated.  The session resource on the CIMI server should be used instead.